keras_cv/metrics/coco/mean_average_precision_test.py

```diff
@@ -22,7 +22,7 @@
 
 
 class COCOMeanAveragePrecisionTest(tf.test.TestCase):
-    def test_runs_inside_model(self):
+    def DISABLE_test_runs_inside_model(self):
```

Comment:
Why are we disabling all these?

Reply:
COCO metrics don't work at all after this PR.  I can make them a private API if we'd like - but I figure until the numerics are worked out we should just rely on the Callback. 

Lets focus on getting the release out first.

Reply:
I'm fine with completely hiding COCOMetrics
However, add TODO at top of the file explaining why we did this, so we won't get lost later